.env.example

```diff
@@ -1,5 +1,6 @@
 MONGOHQ_URL=<THE MONGODB LOCALHOST URL>
 MONGOHQ_URL_TEST=<THE MONGODB LOCALHOST TEST URL>
+LOGIN_URL=<URL OF STAGING OR REVIEW SIGNIN PAGE>
```

Comment:
Is it really necessary to have this env variable? Won't it be better to get/calculate the current url of the app rather than hard coding it?

Reply:
Wouldn't `window.location.origin` or `window.location.href` give you the same information?